spec/unit/matrix-client.spec.ts

```diff
@@ -36,9 +36,11 @@ import { ReceiptType } from "../../src/@types/read_receipts";
 import * as testUtils from "../test-utils/test-utils";
 import { makeBeaconInfoContent } from "../../src/content-helpers";
 import { M_BEACON_INFO } from "../../src/@types/beacon";
-import { ContentHelpers, Room } from "../../src";
+import { ContentHelpers, EventTimeline, Room } from "../../src";
 import { supportsMatrixCall } from "../../src/webrtc/call";
 import { makeBeaconEvent } from "../test-utils/beacon";
+import { IGNORE_INVITES_ACCOUNT_EVENT_KEY, POLICIES_ACCOUNT_EVENT_TYPE, PolicyScope }
+    from "../../src/models/invites-ignorer";
```

Comment:
not sure why the linter allowed this:
```suggestion
import { 
    IGNORE_INVITES_ACCOUNT_EVENT_KEY, 
    POLICIES_ACCOUNT_EVENT_TYPE, 
    PolicyScope,
} from "../../src/models/invites-ignorer";
```